usefulUtilities/sit/sitClient.js

```diff
@@ -73,55 +73,58 @@
 
```

Comment:
We could limit the properties retrieved at line 63 to be more efficient. Also, now that we are able to append this script to any entity to make it a "chair", always placing the avatar at 1.1 times the height of the "chair" object may not always give a realistic seat height. For instance, with a tall tree stump, placing the avatar at 1.1 times the height of the stump could leave a large gap between the top of the stump and the avatar.

Reply:
ACK

Regarding the 1.1x multiplier, you're right. We don't have a better way of doing that right now, and, so far, there haven't been any complaints about such an issue. I will add some text to the Known Issues section of the README about this.